src/app/core/app-routing.module.ts

```diff
@@ -10,6 +10,10 @@ const routes: Routes = [
     path: 'items',
     loadChildren: () => import('../modules/item/item.module').then(m => m.ItemModule)
   },
+  {
+    path: 'yourself',
+    loadChildren: () => import('../modules/yourself/yourself.module').then(m => m.YourselfModule)
+  }
```

Comment:
Users are groups and multiple things in yourself will be group related stuffs. So it should be in the group module and probably under the url `groups/me`.

Reply:
I've moved it to the group module and under that url.